File: CAPITULO 01/Java/ex14.java
import java.util.*;
import java.io.*;

public class ex14{

public static void main(String[]args){
	int idade,ano_atual,ano_nasci,ano_futuro;
	Scanner dado = new Scanner (System.in);
	System.out.println("Digite o ano do seu nascimento: ");
	ano_nasci = dado.nextInt();
	System.out.println("Digite o ano atual : ");
	ano_atual = dado.nextInt();
	idade = ano_atual - ano_nasci;
	ano_futuro = (2050 - ano_atual)+idade;
	System.out.println("Idade atual "+idade+" Idade em 2050 "+ano_futuro);
	}
}

File: CAPITULO 01/C++/ex14.cpp
#include <stdio.h>
int main ()
{
	int idade,ano_atual,ano_nasci,idade_fut;
	printf("Digite o ano de nascimento :");
	scanf ("%d%*c",&ano_nasci);
	printf("Digite a ano atual: ");
	scanf("%d%*c",&ano_atual);
	idade = ano_atual - ano_nasci;
	idade_fut = (2050 - ano_atual) + idade;
	printf("Idade atual = %d idade em 2050 = %d ",idade,idade_fut);
	getchar();
	return 0;
}
